lib/editor/util/validation.js

```diff
@@ -25,6 +25,7 @@ export function doesNotExist (value: any): boolean {
  * Returns false if the value is ok.
  * Returns an EditorValidationIssue object if the value is not ok.
  */
+// eslint-disable-next-line complexity
```

Comment:
Remove that.

Reply:
This is not an error-level eslint rule with our eslint config, however, I think there is a lot of value in having it warn the developer because having functions with tons of complexity will make code very difficult to read and maintain in the future. It's not your fault that this method is bloated in the first place, but it will significantly improve the future readability and maintainability of this code to refactor this method into smaller chunks right now.